src/codegen/utilities/interpreter_helpers.py

```diff
@@ -232,9 +236,11 @@ def create_compound_parameter_request(func):
     return f"grpc_types.{compound_parameter_type}(" + ", ".join(parameters) + ")"
 
 
-def get_response_parameters(output_parameters: list):
+def get_response_parameters(output_parameters: list, is_init_method=False):
```

Comment:
If this function took `func`, you wouldn't have to update its signature to change its behavior.